roles/ceph-common/tasks/main.yml

```diff
@@ -11,6 +11,23 @@
        cache_valid_time=3600
   with_items: ceph.debian_pkgs
 
+- name: create ceph log directory
+  file: dest=/var/log/ceph
+        state=directory
+        owner=root
+        group=adm
+        mode=2750
+
+- name: Change ceph log directory permissions with acl
```

Comment:
I'm thinking not. I think we might be running up against a race condition where the service starts and some of the more frequently written logs are already being written to before the acl is set, but the less frequently written logs are created after the acl is set... The one problem with this is that the group is correct for all logs files. Hit up @sivakom for a more in depth view of what is happening at the deploy. I'm going off of memory from last Thursday or Friday.